applications/TrilinosApplication/custom_utilities/trilinos_cpp_test_utilities.cpp

```diff
@@ -233,25 +234,30 @@ void TrilinosCPPTestUtilities::CheckSparseMatrixFromLocalMatrix(
     values.reserve(total_size);
     for (std::size_t i = 0; i < rB.size1(); ++i) {
         for (std::size_t j = 0; j < rB.size2(); ++j) {
-            row_indexes.push_back(i);
```

Comment:
why are u doing this?? the zeros should be there ,...

Reply:
This is a function to check a sparse matrix from a local matrix, not all values are defined, otherwise will fail the check